### DISCUSSION TITLE
Desktop: Confusing Takeout Upload UX with Albums

### DISCUSSION THREAD

**Discussion by philips** [ente-io/ente#2147] [Enhancements]
It was confusing  to methat when trying to upload from a Google Takeout zip file that ente asks you what album you want to upload to because I feel the ente UX is similar to Google Photos and I would expect it to just end up in my "library". I selected "Uncategorized" which I assume is the equivalent? Regardless, I think the ente UX is confusing here and could use a think.

**Comment by petrst** [comment]
Same confusion here. Will it create albums for each Google Photo albums or all pictures will go to Uncategorized?

**Reply by vishnukvmd** [reply]
Hey, it will create separate albums for each Google Photo album by default.
